src/Employer/Employer.Web/Views/VacancyPreview/VacancyPreview.cshtml

```diff
@@ -55,9 +55,7 @@
                 <p class="govuk-body">Sections you will need to complete:</p>
                 <ul class="govuk-list">
                     <li asp-hide="@Model.HasShortDescription"><a asp-fragment="@Anchors.ShortDescriptionSection" class="govuk-link summary-link">Brief overview</a></li>
-                    <li asp-hide="@Model.HasVacancyDescription"><a asp-fragment="@Anchors.VacancyDescriptionSection" class="govuk-link summary-link">What you will do in your working day</a></li>
-                    <li asp-hide="@Model.HasTrainingDescription"><a asp-fragment="@Anchors.VacancyDescriptionSection" class="govuk-link summary-link">The training you will be getting</a></li>
-                    <li asp-hide="@Model.HasOutcomeDescription"><a asp-fragment="@Anchors.VacancyDescriptionSection" class="govuk-link summary-link">What to expect at the end of your apprenticeship</a></li>
+                    <li asp-hide="@Model.HasVacancyDescription"><a asp-fragment="@Anchors.VacancyDescriptionSection" class="govuk-link summary-link">Description of the apprenticeship</a></li>
```

Comment:
This isn't taking account of TrainingDescription or OutcomeDescription. Maybe we need a new helper property?
```
HasVacancyTrainingOutcomeDescriptions => HasVacancyDescription && HasTrainingDescription && HasOutcomeDescription
```

Reply:
Once you enter the Vacancy Description, you have to enter the Training Description and Outcome Description. That is why just left it as is.